test/e2e_node/swap_test.go

```diff
@@ -42,103 +51,275 @@ const (
 	cgroupV1SwapLimitFile = "/memory/memory.memsw.limit_in_bytes"
 	cgroupV2SwapLimitFile = "memory.swap.max"
 	cgroupV1MemLimitFile  = "/memory/memory.limit_in_bytes"
+	swapUsageCgroupFile   = "memory.swap.current"
 )
 
-var _ = SIGDescribe("Swap", framework.WithNodeConformance(), "[LinuxOnly]", func() {
```

Comment:
why removing `framework.WithNodeConformance()`

Reply:
So @harche and I discussed this, what we want these tests to do is as follows:

1) Run the conformance test suite for sig-node with swap enabled. Leave all defaults as is.
2) Run a few tests to verify that swap is at least functional.

And then we have separate tests that run for the serial jobs that do more involved tests for swap.  

I found that by having all these tests wrapped under a node conformance tag, we were running these jobs in non swap jobs. I figured I'd remove that tag as we can just have the test-infra scripts run the correct jobs with tags.
And then we have more detailed tests to actually test swap but they are run as serial jobs.

Reply:
I'm not sure whether `framework.WithNodeConformance()` = `[NodeConformance]` has any meaningful semantic at this point, so it basically is just another tag that is used by some jobs to decide what to run. If removing it helps run node tests more efficiently, then no objects from me.

Reply:
The important part will be to ensure that we don't loose test coverage - you'll have to keep an eye on this once this merges.